UNRELEASED.md

```diff
@@ -28,7 +28,8 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 - Added accessibility documentation for the account connection and setting toggle components ([#970](https://github.com/Shopify/polaris-react/pull/970))
 - Added accessibility documentation for the avatar component ([#973](https://github.com/Shopify/polaris-react/pull/973))
 - Updated docs about App Bridge usage in AppProvider ([#945](https://github.com/Shopify/polaris-react/pull/945))
-- Added all props example to `DataTable` in the [style guide](https://polaris.shopify.com) ([#1003](https://github.com/Shopify/polaris-react/pull/939))
+- Added all props example to `DataTable` in the [style guide](https://polaris.shopify.com) ([#1003](https://github.com/Shopify/polaris-react/pull/1003))
```

Comment:
Just fix the rebase and wait until the release is done and then it looks good to go. Thank you!